flang/lib/Lower/ConvertVariable.cpp

```diff
@@ -1485,7 +1501,8 @@ lowerExplicitExtents(Fortran::lower::AbstractConverter &converter,
             computeExtent(builder, loc, lowerBounds[spec.index()], ub));
     } else if (spec.value()->ubound().isStar()) {
       // Assumed extent is undefined. Must be provided by user's code.
-      result.emplace_back(builder.create<fir::UndefOp>(loc, idxTy));
+      // result.emplace_back(builder.create<fir::UndefOp>(loc, idxTy));
```

Comment:
Please remove.

Reply:
Removed and updated outdated comments in this file, thanks.